docs/layers/h3-hexagon-layer.md

```diff
@@ -0,0 +1,131 @@
+<!-- INJECT:"PolygonLayerDemo" -->
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/64--bit-support-blue.svg?style=flat-square" alt="64-bit" />
+  <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
```

Comment:
Do we want to add an H3 version badge, similar to what we have on the binding readme: http://github.com/uber/h3-js?

Reply:
I don't expect the JavaScript API to be important here, but the version of the indexing system definitely matters. How stable is the `h3-js` API? Could we relax it to something like ">=2" so users may pick their version?

Reply:
As long as it's the same major version it should be fine. cc @nrabinowitz

Reply:
Honestly even the major version of the library shouldn't matter, as long as it's the same index version.